<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="W1651507" name="Test" tests="2" failures="1" timestamp="28 Apr 2020 20:41:20 GMT" time="37.249" errors="0">
  <testcase name="init" time="9.302" classname="testcases.FacebookLoginTestFail">
    <failure type="java.lang.Exception" message="Contact number error!!!">
      <![CDATA[java.lang.Exception: Contact number error!!!
	at pages.FacebookLoginPage.assertLoginPageErrorMsg(FacebookLoginPage.java:139)
	at testcases.FacebookLoginTestFail.init(FacebookLoginTestFail.java:22)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- init -->
  <testcase name="init" time="8.526" classname="testcases.FacebookLoginTestPass"/>
</testsuite> <!-- Test -->
